### PR TITLE
Add back English keyboard

### PR DESCRIPTION
**Pull request by Jag-Marcel**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

This PR is meant to add the English keyboard back into Scribe, in preparation for 3.2. With #476, the translation function would also make sense now.

### Related issues

- #7

### CODE REVIEW

**Comment by andrewtavis** [comment]
Alright @Jag-Marcel :) Sorry that this took so long. You now have an English database on this branch 😊 It has the verbs, nouns and autosuggestions, but not anything else for now. We can run the process to make the emojis at some point before the release!

I've set aside some time in the sync to go through the minor changes for the verbs that I made while I was redoing the formatting process. Let me know if you have any questions!

**Comment by andrewtavis** [comment]
Note also @Jag-Marcel that I've mapped out all of the conjugations in the table within the [scribe-gsoc-sync-2024-8-3](https://etherpad.wikimedia.org/p/scribe-gsoc-sync-2024-8-3) notes 😊 Hope that's helpful to know what's going on!

**Comment by andrewtavis** [comment]
FYI @Jag-Marcel you also now have the emoji data for English, so everything should be functional except for translation. Just need to clean up the code to generate the emojis, and then next translations 😊

**Comment by Jag-Marcel** [comment]
@andrewtavis I think something got messed up with that last commit, I tried merging in the main branch to see if that fixed it but it's giving me the same results.
When I try to pull the changes, it's giving me these errors:
```
error: unable to read sha1 file of Keyboards/LanguageKeyboards/Italian/ITLanguageData.sqlite (77f0c8935af7180ebff4f38f4555c5860859beb8)
error: unable to read sha1 file of Keyboards/LanguageKeyboards/Portuguese/PTLanguageData.sqlite (971750e0ec034cf5e28508a4e160d2f9f2406361)
error: unable to read sha1 file of Keyboards/LanguageKeyboards/Russian/RULanguageData.sqlite (05cca6a4fcb107f26f69e766ce85769c9a1d17a6)
error: unable to read sha1 file of Keyboards/LanguageKeyboards/Spanish/ESLanguageData.sqlite (52c71cf704633d82ecfd2a72b98036ae7b410a49)
error: unable to read sha1 file of Keyboards/LanguageKeyboards/Swedish/SVLanguageData.sqlite (1a51208f893ca8a95bd4f507507a948f1f6a1914)
Updating 3e568ff..da5d357
```
In the commit itself it also seems most data got deleted:

image

**Comment by andrewtavis** [comment]
I'm away from my personal computer, @Jag-Marcel, but if we just need the data back in we should be able to figure this out :) 

Steps to follow:
- Make sure that your local copies of Scribe-iOS and Scribe-Data are in the same directory, ideally scribe-org as maybe the transfer script's a bit weird
- Pull Scribe-Data and make sure that Scribe-iOS is on your branch
- Run the command in [Scribe-Data/src/scribe_data/load/send_dbs_to_scribe.py](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/load/send_dbs_to_scribe.py)
- You'll then have the databases on your Scribe-iOS branch and can recommit them

Hopefully this is the full steps. I'll check in on Matrix as the notifications will come through there 😊

**Comment by andrewtavis** [comment]
4c8fb19 makes the necessary changes to allow for setting the period and comma on the letter keys, @Jag-Marcel :) One thing to note, for conjugation it would be great if we'd present both options that are available in a following 1x2 view. You can see this in the case declension views specifically where when you press `Dat` after typing `mit` you'll see both options are at times shown to the user, which then indicates to them that they'll need to make a change. As of now when I conjugate `Go` I just see `Go` as the present form instead of `Go / Goes`. Once we have this in I'll give it one final review and then we'll be good to merge!

**Comment by Jag-Marcel** [comment]
@andrewtavis does declension even work in the current state? I don't seem to get anything for non-verbs in the conjugation command, and when I tried pressing on `mit` in the autocompletions the German keyboard crashed.

**Comment by andrewtavis** [comment]
There is some combination of declension and conjugation where some functions are shared and some aren't. Maybe one of the recent changes broke it? I didn't do the best job of reviewing as I forgot to check that declension was still functioning.

**Comment by Jag-Marcel** [comment]
@andrewtavis I'll look into it, but I also found another issue: The past perfect simple forms use the -ing forms of verbs instead of the participle (instead of "gone" in this example)
image
Only the past forms though.

**Comment by Jag-Marcel** [comment]
Ok, declensions work now, it was the same issue I had earlier with emoji autosuggestions ([`26a0caa`](https://github.com/scribe-org/Scribe-iOS/pull/484/commits/26a0caaf5ae4fb3d4c300f79f45ace9553810c7a)) the columns in the sqlite tables changed column names at some point and that wasn't reflected in the code yet

**Comment by andrewtavis** [comment]
Fantastic, @Jag-Marcel, and ya I saw that that verb conjugation's all screwed up... Let me fix that really quick and you can pull in the new one :)

**Comment by andrewtavis** [comment]
Ok, @Jag-Marcel, Scribe-Data now has the updated English DB with the correct version of the verbs :) Let me know if anything else is needed! Once we have the English keyboard in it's full speed ahead on finishing up cross language translation! Really appreciate all the efforts here 🙏😊

**Comment by Jag-Marcel** [comment]
Alright, that should be everything now 👍

**Comment by andrewtavis** [comment]
🔥🔥

Thanks @Jag-Marcel! I'll review this tonight :) 

Do you want to hop over to the i18n changes now? Also let me know on Matrix if there's anything else you'd like to get done in the next week+ aside from the i18n refactor and translations 😊

File: Keyboards/KeyboardsBase/LanguageDBManager.swift [hunk 1]
```diff
@@ -250,7 +250,7 @@ extension LanguageDBManager {
       word = ?
     """
     let args = [word]
-    let outputCols = ["suggestion_0", "suggestion_1", "suggestion_2"]
+    let outputCols = ["autosuggestion_0", "autosuggestion_1", "autosuggestion_2"]
```
Comment (andrewtavis) [inline Keyboards/KeyboardsBase/LanguageDBManager.swift:253]: Nice catch on these! I honestly don't remember when the change for this was made 🤔 Maybe is coming from changes in Scribe-Data's directory structure or something like that :)

**Comment by andrewtavis** [comment]
dc554ed puts a space between conjugation table values for English and the participle where in some cases before there was just a slash. I think that we're fine with a space and the user will understand that the participle is just there for a backup so they don't need to do the command again. It also fixes a minor bug I found: That if you entered a capitalized plural word to the plural command it would show "Already plural" as expected, but then the returned word would be lower case as we lower cased it in the Plural command code to make sure it matches on the value in the DB. Something that hadn't been noticed when testing in German for obvious reasons 😇

**Review by andrewtavis — APPROVED** [review]
Thanks so much for the word to get issue number 🥁🥁🥁🥁 7 done, @Jag-Marcel! Totally amazing that this is on the way :) We'll need to update the media for the App Store so that we indicate that there are eight keyboards instead of seven 😊 I'll make that issue now.

CC @wkyoshida and @SaurabhJamadagni 🙌

**Pull request merged**